+bids/layout.m

```diff
@@ -110,10 +110,10 @@
 for su=1:numel(sub)
     sess = cellstr(file_utils('List',fullfile(BIDS.dir,sub{su}),'dir','^ses-.*$'));    
     for se=1:numel(sess)
-        if isempty(BIDS.subjects)
-            BIDS.subjects = parse_subject(BIDS.dir, sub{su}, sess{se});
+        if isempty(BIDS.subses)
```

Comment:
What do you think about using `subjs` instead of `subses`? In Matlab, `subs` has a somewhat conventional meaning of "subscripts" (that is, a multidimensional indexing vector), and "subses" could be interpreted as "an array of subscript arrays".

Reply:
Oh, wait: I missed the part about this now meaning "subjects *and* sessions". Sorry.